assets-and-images.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: page
+title: Assets and Images
+permalink: /assets-and-images/
+---
+
+Flutter applications include both code and assets (sometimes called resources).
+Assets can include static data, configuration files, or anything else an
+application needs to function; in particular, assets include icons and other
+images displayed in the UI.
+
+## Specifying assets
+
+The flutter build process uses the `flutter.yaml` file to identify assets
```

Comment:
can we link to what the "build process" is? If not, maybe just say "Flutter uses the ...", which helps avoid introducing a new (undefined) concept.